ftw/simplelayout/contenttypes/upgrades/20200317204342_remove_obsolete_videoblock_js_bundle/upgrade.py

```diff
@@ -0,0 +1,9 @@
+from ftw.upgrade import UpgradeStep
+
+
+class RemoveObsoleteVideoblockJsBundle(UpgradeStep):
+    """Remove obsolete videoblock js bundle.
+    """
+
+    def __call__(self):
+        self.install_upgrade_profile()
```

Comment:
@Nachtalb There should be a condition right? To only apply this change for plone5?

Reply:
Oh yeah you're right..